src/components/admonition/examples/admonition-custom-heading.tsx

```diff
@@ -0,0 +1,29 @@
+import { Component, h } from '@stencil/core';
+
+/**
+ * With custom `heading`
+ *
+ * By default, the title will equal the `type` qualifier.
+ * However, it is possible to use a `type` just to get the desired visualisation
+ * (icon and color), but override the default heading, using the `heading` prop.
+ */
+@Component({
+    tag: 'limel-example-admonition-custom-heading',
+    shadow: true,
+})
+export class AdmonitionCustomTitleExample {
+    public render() {
+        const description = `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum viverra magna non pretium condimentum. Integer a nunc magna. In commodo elit turpis, porttitor vulputate odio pretium a. Ut at sapien a massa convallis commodo eu sed ligula. Curabitur non sodales neque. Sed id nisl vel ex tempor euismod. Nulla nec lorem dui. Cras tincidunt urna nec velit pretium maximus.
+
+        Praesent sed cursus lorem. Phasellus lobortis dolor vitae pretium bibendum. Vivamus non augue in urna consequat dapibus at quis diam. Duis tristique lacinia felis, quis condimentum urna interdum sit amet. Suspendisse facilisis pulvinar suscipit. Pellentesque quis velit feugiat, bibendum erat at, ornare tortor. Nullam sed risus a enim tempor eleifend nec quis nulla. Cras quis pellentesque justo. Maecenas id justo a eros consequat auctor.
```

Comment:
Even though this is a good example and shows a smart hack to get line breaks, I'd still argue that this component without Markdown support is waaaaay too weak really. For my taste, we are below an MVP level.

We can lunch it this way, but as long as adding markdown support does not add a BREAKING CHANGE. I can't assess that of course, and don't know how much time or effort is needed add this support.

A breaking change could be for instance that we think it's better to call this prop `content` instead of `description`, since we now have markdown support. (I already think `description` is a bad prop name actually.)

As soon as we lunch this, I'm sure users will soon want to add bullet point list, links, etc… in the text.